Formula/bitcoin.rb

```diff
@@ -11,6 +11,7 @@ class Bitcoin < Formula
     patch do
       url "https://github.com/bitcoin/bitcoin/commit/1ec0c0a01c31.patch?full_index=1"
       sha256 "a1f761fe29f78e783cb4b55f8029900f94b45d1188cb81c80f73347ee2fdc025"
+      revision 1
```

Comment:
@ilovezfs `revision` inside this `patch` block, or outside?

Reply:
should go under homepage in this case